src/base.py

```diff
@@ -37,9 +37,9 @@ def change_to_base(n: int, b: int) -> str:
     n *= tegn         # because flooring work by rounding to the lowest number we have a problem when negative, i've tried to remove that by making n positive for now
     liste = []         #a list for our results
     while n:
-        liste.append(int(n%b)) #making n in our base
+        liste.append(str(n%b)) #making n in our base
```

Comment:
This will work when `n%b` is less than 10, which it will be when `b <= 10`, but it will fail if you try to to convert to, for example, base 16. There `n = 15` should be `"A"`, but you will get `"15"`